packages/providers/src/operations.graphql

```diff
@@ -44,15 +44,11 @@ fragment transactionFragment on Transaction {
 fragment receiptFragment on Receipt {
   contract {
     id
-    bytecode
-    salt
   }
```

Comment:
I'm unsure if we should remove it _from_ the `operations.graphql`. 🤔 

We may be able to opt out of these fields for the dry-run calls _(on a per-call basis)_.